core/test/Store.js

```diff
@@ -16,6 +16,7 @@ contract("Store", function(accounts) {
   const erc20TokenOwner = accounts[2];
 
   const identifier = web3.utils.utf8ToHex("id");
+  const tokenAddr = "0x249bB6CF1751c221058749a9B571F1Fb0a621443";
```

Comment:
Is this an arbitrary address? If so, prefer creating it this way so it's clear that the exact address doesn't matter:
```
const arbitraryTokenAddress = web3.utils.randomHex(20);
```